qtt/instrument_drivers/virtualAwg/virtual_awg.py

```diff
@@ -420,7 +420,9 @@ def sequence_gates(self, sequences, do_upload=True):
             upload_data.append((sequence_names, sequence_channels, sequence_items))
             if do_upload and sequence_items:
                 self.awgs[number].upload_waveforms(sequence_names, sequence_channels, sequence_items)
-        return {'gate_comb': sequences, 'upload_data': upload_data, 'settings': settings_data}
+                
+        self._sequence_data = {'gate_comb': sequences, 'upload_data': upload_data, 'settings': settings_data}                        
```

Comment:
OK???

Reply:
After performing an action with the virtual awg (e.g. `sweep_gates`) I want to be able te retrieve the data that was uploaded (for debugging). There data is returned by the `sequence_gates`, but the calling method does not always store that data. Would it help if I create a public method `virtual_awg.return_latest_sequence_data()`?

@CumulonimbusCalvus